rustls/Cargo.toml

```diff
@@ -10,8 +10,12 @@ homepage = "https://github.com/rustls/rustls"
 repository = "https://github.com/rustls/rustls"
 categories = ["network-programming", "cryptography"]
 autobenches = false
+build = "build.rs"
```

Comment:
Why not use the rustversion attribute directly as discussed in #951?

Reply:
> Why not use the rustversion attribute directly as discussed in #951?

This is explained in build.rs.